action/protocol/poll/nativestaking.go

```diff
@@ -94,55 +94,60 @@ func (ns *NativeStaking) Votes() (*VoteTally, time.Time, error) {
 		Buckets:    make([]*types.Bucket, 0),
 	}
 	prevIndex := big.NewInt(0)
-	limit := big.NewInt(256)
+	limit := big.NewInt(160)
 
 	for {
-		vote, err := ns.readBuckets(prevIndex, limit)
+		vote, index, err := ns.readBuckets(prevIndex, limit, correctGas)
```

Comment:
Can we test the case when we have more than 160 buckets? So we can make sure pagination is correct?

Reply:
yes, i tested last night pagination is correct

we now have 161 buckets, i read using limit = 256 (result in 1 batch) vs. limit = 160 (result in 2 batch), the final vote count is same

Reply:
let's add a unit test to reflect this case

Reply:
added in latest push